examples/sklearn/iris-classifier/README.md

```diff
@@ -51,35 +51,36 @@ $ python3 trainer.py
 ## Implement a predictor
 
 1. Create another Python file `predictor.py`.
-2. Add code to load and initialize your pickled model.
-3. Add a prediction function that will accept a payload and return a prediction from your model.
+2. Define a Predictor class with a constructor that loads and initializes your pickled model.
+3. Add a predict function that will accept a payload and return a prediction from your model.
 
 ```python
 # predictor.py
 
+import boto3
 import pickle
-import numpy as np
-
+import re
 
-model = None
-labels = ["setosa", "versicolor", "virginica"]
 
+class Predictor:
+    def __init__(self, config):
+        bucket, key = re.match("s3://(.+?)/(.+)", config["model"]).groups()
+        s3 = boto3.client("s3")
+        s3.download_file(bucket, key, "model.pkl")
 
-def init(model_path, metadata):
-    global model
-    model = pickle.load(open(model_path, "rb"))
+        self.model = pickle.load(open("model.pkl", "rb"))
+        self.labels = ["setosa", "versicolor", "virginica"]
 
+    def predict(self, payload):
+        measurements = [
+            payload["sepal_length"],
+            payload["sepal_width"],
+            payload["petal_length"],
+            payload["petal_width"],
+        ]
 
-def predict(payload, metadata):
-    measurements = [
-        payload["sepal_length"],
-        payload["sepal_width"],
-        payload["petal_length"],
-        payload["petal_width"],
-    ]
-
-    label_id = model.predict(np.array([measurements]))[0]
-    return labels[label_id]
+        label_id = self.model.predict(np.array([measurements]))[0]
```

Comment:
If I'm following the diff correctly, is np no longer imported but still used here?